07-ML basics/07-ml.md

```diff
@@ -30,16 +35,21 @@ read more about this subject.
 -   Supervised and Unsupervised Learning
 -   Dataset
 -   Problem types - Regression, Classification, Clusterization, Anomaly Detection.
--   Tabular Data
 -   Linear Regression
 -   Decision Tree
 -   Neural network
 -   CNN
+-   RNN
+-   Style Transfer
 -   Ensemble
 -   Model Metrics - Accuracy, F1, etc...
--   Tensorflow vs Pytorch
+-   Tensorflow & Pytorch as Machine Learning Libraries
 -   Data Pre Processing
 
 ### Final Exercise:
-Create a model that predicts avocado prices. You can find the dataset in this folder.
-Try use everything you have learned to make the most accurate model you can!
\ No newline at end of file
+Try use everything you have learned to make the most accurate avocado price prediction model you can!
+
+- Create a model that predicts avocado prices, using the dataset in this folder.
```

Comment:
The dataset exists in the TS, explain how a trainee should access it, and that the training of the model should happen on Run:AI

Reply:
I added this in the Hafifa Templates